src/main/java/blackjack/domain/Dealer.java

```diff
@@ -0,0 +1,28 @@
+package blackjack.domain;
+
+import java.util.List;
+
+public class Dealer extends User {
+    private static final int DEALER_CRITICAL_SCORE = 16;
+    private static final int COUNT_OF_INITIAL_OPEN_CARDS = 1;
+    private static final String KOREAN_NAME = "딜러";
+
+    public Dealer() {
+        this.name = KOREAN_NAME;
+    }
+
+    public static int getCriticalScore() {
+        return DEALER_CRITICAL_SCORE;
+    }
+
+    @Override
+    public List<Card> getInitialCards() {
+        return cards.getCards()
```

Comment:
getCards를 꺼내서 subList하지말고 Card일급컬렉션인 Cards에서 subList를 지원하도록 해주세요
```suggestion
        return cards.subList(START_INDEX, COUNT_OF_INITIAL_OPEN_CARDS);
```

Reply:
적용했습니다!